docs/products/cassandra/get-started.rst

```diff
@@ -0,0 +1,88 @@
+
+**Projects** allows you to organize groups of services under different topics and each project can even have different billing settings. An empty project is automatically created for you upon sign up and the free credits are attached to this project. 
+
+You can create new projects by clicking the project dropdown at the top of the sidebar and selecting "Create a new project". The same menu can also be used to switch between projects.
+
+To get started with Cassandra, first click the "Create a new service" button.
+
+.. image:: /images/products/cassandra/cassandra-console.png
+
+The dialog that opens allows you to specify the main service properties:
+
+* **Service name**: A short name for the service used to distinguish it from others. A random name is provided, but you can type in a more user-friendly one.
+
+* **Service type**: Select "Cassandra".
+
+* **Plan**: How many servers and what kind of memory/CPU/disk resources will be allocated to run your service.
+
+* **Cloud**: Which cloud and region to run the service on. *Note*: the pricing of the same service may differ between cloud providers and their regions.
+
+After selecting your service properties, click the "Create" button and you will be taken back to the service list view displaying the newly selected service with an indicator that it is being created.
+
+Click the service name in the list and the service's "Overview" information page will open. Aside from displaying the service's connection parameters and its status, this view is where you can make changes to the service.
+
+The "Status" indicator will say "REBUILDING" while the service is being created for you. Once the service is up and running, the light will change to green and it will say "RUNNING". Note: while services typically start in minutes, the performance between clouds varies and it can take longer in some cases.
+
+.. image:: /images/products/cassandra/cassandra-console-cluster.png
+
+The Cassandra cluster is initiated with a superuser role named avnadmin. The role can be used to create keyspaces and additional roles with any tool or library supporting the Cassandra protocol.
+
+Data center and replication
+---------------------------
+
+Your Cassandra cluster is created with all nodes in a Data Center (DC) named aiven. It's strongly recommended to use the NetworkTopologyStrategy replication strategy when creating keyspaces to ensure replicas of your data are distributed on nodes in different Availability Zones (AZ) within the service's selected cloud. 
+
+While SimpleStrategy has a replication factor property, it may place the replicas in the same AZ. The examples below demonstrate how to create keyspaces with NetworkTopologyStrategy using the aiven DC.
```

Comment:
```suggestion
While ``SimpleStrategy`` has a replication factor property, it may place the replicas in the same AZ. The examples below demonstrate how to create keyspaces with ``NetworkTopologyStrategy`` using the aiven DC.
```